packages/patternfly-4/react-charts/src/components/ChartLegend/ChartLegend.js

```diff
@@ -12,6 +12,22 @@ export const propTypes = {
    * See TypeScript API docs: https://formidable.com/open-source/victory/docs/victory-legend/
    */
   '': PropTypes.any,
+  /**
+   * The containerComponent prop takes an entire component which will be used to
+   * create a container element for standalone charts.
+   * The new element created from the passed containerComponent wil be provided with
+   * these props from VictoryArea: height, width, children
+   * (the chart itself) and style. Props that are not provided by the
+   * child chart component include title and desc, both of which
+   * are intended to add accessibility to Victory components. The more descriptive these props
+   * are, the more accessible your data will be for people using screen readers.
+   * Any of these props may be overridden by passing in props to the supplied component,
+   * or modified or ignored within the custom component itself. If a dataComponent is
+   * not provided, VictoryArea will use the default VictoryContainer component.
+   * @example <ChartContainer title="Chart of Dog Breeds" desc="This chart shows how popular each dog breed is by percentage in Seattle." />
+   * @default <ChartContainer/>
+   */
+  containerComponent: React.ReactElement,
```

Comment:
`React.ReactElement` needs a type (like `React.ReactElement<any>`)

Reply:
That doesn't compile

![Screen Shot 2019-05-29 at 1 08 30 PM](https://user-images.githubusercontent.com/17481322/58576679-e258bf80-8212-11e9-9c52-cc2809085c7e.png)

Reply:
Replaced React.ReactElement with PropTypes.element

Reply:
Ah, I didn't even realize this was JS instead of TS. Yes, PropTypes.element is correct.